theme/less/homepage.less

```diff
@@ -20,6 +20,26 @@ main article
   column-gap: 20px;
 }
 
+main article ol
+{
+  line-height: 1.5rem;
```

Comment:
Suggestion:
```
main article ol
{
  line-height: 1.4rem;
}

article
{
  font-size: 0.85rem;
}
```

Reply:
If you like content being 0.9rem more on other pages apply font-size change only on homepage.

Reply:
Can you check please this values one more time? This is the resault ->
![image](https://user-images.githubusercontent.com/19854505/89305379-9cea1a80-d67f-11ea-8b3d-5afc30e67cd7.png)

Reply:
Seems like you are still using `0.9rem` instead of `0.85rem`, but I thinking keeping `0.9rem` is fine, now after seeing all other parts being more aligned with the Drupal version.

Reply:
True. But 0.92 gives result similar to Drupal version.

Reply:
Calculated line height in Drupal website is `21px`, in CMintS version is `22.4px`, Increasing line height can't give similar result, unless you are planing to adjust height by changing `line-heigh` which is not a good idea I guess.